wrapper/src/main/java/software/amazon/jdbc/util/RdsUtils.java

```diff
@@ -372,6 +425,8 @@ public RdsUrlType identifyRdsType(final String host) {
       return RdsUrlType.RDS_AURORA_LIMITLESS_DB_SHARD_GROUP;
     } else if (isRdsProxyDns(host)) {
       return RdsUrlType.RDS_PROXY;
+    } else if (isRdsProxyEndpointDns(host)) {
+      return RdsUrlType.RDS_PROXY_ENDPOINT;
```

Comment:
Curious why do we need these? Do the existing `isRdsProxyDns` and `RDS_PROXY` not cover the same cases? Could we remove one of these?

Reply:
Some of our plugins are not compatible with RDS Proxy so it's important to properly recognize all type of urls related to RDS Proxy. we can reuse the same RDS_PROXY but I think it's better to be precise.